src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -4443,12 +4441,15 @@ class StubGenerator: public StubCodeGenerator {
     __ movptr(key, key_mem);
     __ movptr(state, state_mem);
 #endif
+    __ subptr(rsp, 96 * longSize); // Create space on the stack for htbl entries
```

Comment:
Is this aligned correctly? Or alignment does not matter?

Reply:
Thanks, Vladimir. I've addressed both your comments in the latest update.